### PR TITLE
Update CI files

### PR DESCRIPTION
**Pull request by pulpbot**
[noissue]

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]

File: template_config.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ docker_fixtures: true
 docs_test: true
 flake8: true
 issue_tracker: redmine
+noissue_marker: '[noissue]'
```
Comment (quba42) [inline template_config.yml:26]: What would be an alternate noissue marker?

Comment (mdellweg) [reply template_config.yml:26]: https://github.com/ansible/galaxy_ng/pull/883/files#diff-48fd5085612376b649898c2408233a202481ac724dd377c1369cf20ee34649ddR41

**Comment by mdellweg** [comment]
Test are failing, because this import should be changed to `from pulp_smash.pulp3.bindings import delete_orphans`:
https://github.com/pulp/pulp_deb/blob/main/pulp_deb/tests/functional/api/test_crud_content_unit.py#L9

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
> Test are failing, because this import should be changed to `from pulp_smash.pulp3.bindings import delete_orphans`:
> https://github.com/pulp/pulp_deb/blob/main/pulp_deb/tests/functional/api/test_crud_content_unit.py#L9

Should be solved by #333

**Comment by quba42** [comment]
#333 Was merged. This PR should turn green again by tomorrow then.

File: template_config.yml (hunk 1 above)
Comment (quba42) [reply template_config.yml:26]: Interesting.

**Pull request merged**